src/strategies/frax-finance/index.ts

```diff
@@ -5,13 +5,11 @@ import { multicall } from '../../utils';
 const BIG6 = BigNumber.from('1000000');
 const BIG18 = BigNumber.from('1000000000000000000');
 
-const UNISWAP_SUBGRAPH_URL = {
-  '1': 'https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v2'
-};
-
 export const author = 'FraxFinance';
 export const version = '0.0.1';
```

Comment:
would be good to raise this version so others can understand there is a change, also if you have some time would be nice to have `frax-finance` strategy changes in a separate PR

Reply:
I consolidated into one strategy in 986a603